models/articles-models.js

```diff
@@ -14,4 +14,19 @@ function selectArticleById(article_id) {
     });
 }
 
-module.exports = {selectArticleById};
+function selectArticles() {
+  return db
+    .query(
+      `
+  SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments.article_id) AS comment_count FROM articles
```

Comment:
Good - where is article_img_url being selected from? I'd specify this the same way as you've done so for the other columns for clarity